pyproject.toml

```diff
@@ -205,6 +204,10 @@ warn_return_any = true
 warn_unreachable = true
 warn_unused_ignores = true
 
+[[tool.mypy.overrides]]
```

Comment:
Could you please add a comment explaining why it is here (because of docs)?

Reply:
Done